libsolutil/CommonIO.cpp

```diff
@@ -79,6 +79,17 @@ string solidity::util::readUntilEnd(istream& _stdin)
 	return ss.str();
 }
 
+string solidity::util::readBytes(istream& _input, size_t _length)
+{
+	string output;
+	output.resize(_length);
+	_input.read(output.data(), static_cast<streamsize>(_length));
+	// If read() reads fewer bytes it sets falibit.
```

Comment:
Typo. Also I think EOF should be mentioned in some way to make it understandable.
```suggestion
	// If read() reads fewer bytes it sets failbit in addition to eofbit.
```